nengo_sphinx_theme/theme/sidebar.html

```diff
@@ -1,17 +1,56 @@
-<div role="complementary" class="bs-docs-sidebar hidden-print" id="nav-affix">
-  <ul class="bs-docs-sidenav bs-sidenav nav" role="complementary">
+<a class="toggle-sidenav d-block d-md-none" href="#"
+  ><i class="icon-close fa fa-fw fa-arrow-left"></i
+  ><i class="icon-open fa fa-fw fa-arrow-right"></i
+></a>
+<div role="complementary" class="sidenav col-4 col-xl-3 p-0 border-right">
+  <h3 class="pt-5 px-5">
     <a href="{{ pathto(master_doc) }}">
       {%- if theme_nengo_logo %}
-      <img class="logo" src="https://www.nengo.ai/design/_images/{{ theme_nengo_logo }}" alt="{{ project }}" width="{{ theme_sidebar_logo_width|toint }}" />
-      {%- elif logo %}
-      <img class="logo" src="{{ pathto('_static/' + logo, 1) }}" alt="{{ project }}" width="{{ theme_sidebar_logo_width|toint }}" />
```

Comment:
Did the styling change so that setting the logo width is no longer necessary? I think I remember we did this so that projects with different length logos would still have uniform widths in the sidebar.

Reply:
Yeah I think what happens now is that the height is fixed rather than the making the width modifiable. The main reason we would want that to be different is if some projects didn't want to use the full logo, just wanted to use the square logo (which is what [SPA does](https://www.nengo.ai/nengo-spa/)), but it's another thing where I think it's better to just be consistent across all projects.

Reply:
Did some testing, and it looks like the logo dynamically resizes to fit the size of the sidebar. I was testing with the nengo-spinnaker logo, which is the biggest, and everything looked good to me for various window sizes, so I think this is all good.